examples/PDSCHLTEExample.m

```diff
@@ -0,0 +1,573 @@
+%% PDSCH Throughput Conformance Test for Single Antenna (TM1), Transmit Diversity (TM2), Open Loop (TM3) and Closed Loop (TM4/6) Spatial Multiplexing
+% This example demonstrates how to measure the Physical Downlink Shared
+% Channel (PDSCH) throughput performance using LTE Toolbox(TM) for the
+% following transmission modes (TM):
+%
+% * TM1: Single antenna (Port 0)
+% * TM2: Transmit diversity
+% * TM3: Open loop codebook based precoding: Cyclic Delay Diversity (CDD)
+% * TM4: Closed loop codebook based spatial multiplexing
+% * TM6: Single layer closed loop codebook based spatial multiplexing
+%
+% The example also shows how to parameterize and customize the settings for
+% the different TMs. It also supports the use of 
+% Parallel Computing Toolbox(TM) to reduce effective simulation time.
+
+% Copyright 2016-2018 The MathWorks, Inc.
```

Comment:
How come we are adding a matlab test script under examples? This uses none of the functions we've build and is available for anyone with matlab access anyway.

Reply:
Where did this script come from? 🤣 Who commited it?

Reply:
I think I accidentally did when I tried to figure out what was wrong. I'll just remove it.